api-service/src/controllers/ListQueryTemplates/ListTemplateValidationSchema.json

```diff
@@ -0,0 +1,88 @@
+{
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+            "enum": [
+                "api.query.template.list"
+            ]
+        },
+        "ver": {
+            "type": "string"
+        },
+        "ts": {
+            "type": "string"
+        },
+        "params": {
+            "type": "object",
+            "properties": {
+                "msgid": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "msgid"
+            ],
+            "additionalProperties": false
+        },
+        "request": {
+            "type": "object",
+            "properties": {
+                "filters": {
+                    "type": "object",
+                    "properties": {
+                        "query_type": {
+                            "type": "string",
+                            "enum": [
+                                "json",
+                                "sql"
+                            ]
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "sortBy": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "field": {
+                                "type": "string",
+                                "enum": [
+                                    "created_date",
+                                    "updated_date"
+                                ]
+                            },
+                            "order": {
+                                "type": "string",
+                                "enum": [
+                                    "asc",
+                                    "desc"
+                                ]
+                            }
+                        },
+                        "additionalProperties": false,
+                        "required": [
+                            "field",
+                            "order"
+                        ]
+                    }
+                },
+                "limit": {
```

Comment:
where is the min and max value for limit and offset

Reply:
done